Split the 'Additional scripts' bonus guide into three bonus guides: Aliases, lnbalance and lnchannels (new PR without conflicts)

VajraOfIndra opened the pull request:
Identical to #841 but without merging conflicts.

Stadicus commented:
Seems like the following two images are missing:

* images/alias-example.png
* images/script-lnchannels.png

Otherwise, all seems in order. Although I didn't do a proper diff... :)

RooSoft commented on bonus/raspberry-pi/aliases.md:
```diff
@@ -0,0 +1,189 @@
+  alias showversion='echo The installed versions of the services are as follows: ; \
+    bitcoind --version ; \
+    lnd --version ; \
+    echo "BTC RPC Explorer: $(jq '.version' < /home/btcrpcexplorer/btc-rpc-explorer/package.json)" ; \
```
The `/home/btcrpcexplorer` folder doesn't exist on my raspibolt... am I missing something?

RooSoft replied:
Oh, the alias requires some btc explorer to be installed... might be a good idea to explain dependencies those alias have...

Stadicus replied:
Good point, if some application is not installed, an error is thrown. This can be avoided by at least piping the error message into nirvana (/dev/null):
```
cat /home/btcrpcexplorer/btc-rpc-explorer/package.json 2>/dev/null | jq '.version
```

VajraOfIndra replied:
I added this solution to the Explorer and RTL lines. For Electrs, not sure how to do it so I'll let ou add it as a separate PR if you wish so.

RooSoft reviewed (approved):
All good to me aside from a minor comment that could end up creating a different PR...

RooSoft commented:
> Seems like the following two images are missing:
> 
> * images/alias-example.png
> * images/script-lnchannels.png
> 
> Otherwise, all seems in order. Although I didn't do a proper diff... :)

Good catch... I concur.

VajraOfIndra commented:
> Seems like the following two images are missing:
> 
>     * images/alias-example.png
> 
>     * images/script-lnchannels.png
> 
> 
> Otherwise, all seems in order. Although I didn't do a proper diff... :)

Good catch! I added them. I also fixed the path for the picture for lnbalance.

VajraOfIndra commented:
Thanks all for the reviews, I'm now merging the PR!
